contributing/code/stack_trace.rst

```diff
@@ -0,0 +1,183 @@
+Getting a Stack Trace
+=====================
+
+When :doc:`reporting a bug </contributing/code/bugs>` for an
+exception or a wrong behavior in code, it is crucial that you provide
```

Comment:
It's not really all or nothing, what I want to convey here is that it can really make a big difference, and more often than not, it really does. For instance I would have a really hard time understanding the bug you reported without it. I'd rather have exceptions systematically reported with a stack trace rather than the other way around, and I believe a lot of maintainers would agree.